test/support/helpers.ex

```diff
@@ -3,7 +3,7 @@ defmodule Formulator.Test.Helpers do
   alias Formulator.Test.SampleSchema
 
   def prepare_form(attrs) do
-    %Form{data: attrs}
+    %Form{id: :fake, data: attrs}
```

Comment:
Yes, this is a mystery guest. Our tests rely on this name when verifying the form fields. In a normal phoenix application this would either be the struct name if you passed in a changeset or the `as` option if you passed in the `conn`.

Given your comment, I think I'd like to eliminate that mystery guest, what do you think about:

```elixir
# test
form = prepare_form(attrs, name: :fake)

# helper
def prepare_form(attrs, [name: name]) do
````

Maybe swap the order of the options, don't have a strong opinion there.

Reply:
Yeah, I think what you suggest would be an improvement. I took a look at the code that is exercising the `prepare_form/1` above, and it looks like this, 

```elixir
input =
  %{name: ""}
  |> prepare_form
  |> Formulator.input(:last_name, label: false)	
  |> extract_html
```

could you use the `%{name: ""}` there or do you have to pass a separate keyword args?

Reply:
`%{name: ""}` in this case is the attributes that are being passed in so we'd nee to pass in an argument as the section option.

```elixir
input =
  %{name: ""}
  |> prepare_form(form_name: :user)
  |> Formulator.input(:last_name, label: false)	
  |> extract_html
```

Reply:
Yeah, I like that last one. I like that you called it `form_name` instead of just `name`. It removes some of the confusion with the `%{name: ""}` attributes.